example/ctc/lstm_ocr.py

```diff
@@ -59,7 +59,7 @@ def gen_rand():
 def get_label(buf):
     ret = np.zeros(4)
     for i in range(len(buf)):
-        ret[i] = 1 + int(buf[i])
+        ret[i] = int(buf[i])
```

Comment:
contrib.ctcloss has been there for a while. We should maintain backward compatibility

Reply:
Making a 1-indexed API into 0-indexed API is not really backward compatible. Contrib package doesn't guarantee backward compatibility anyway.

Reply:
My understanding is that zero is reserved for CTC blank label, then we shall not use zero as label (which is ensured by the old code).

Reply:
With this change, alphabet_size is reserved for CTC blank label.